src/olav_espenes_ex/ex05/myrand.py

```diff
@@ -0,0 +1,48 @@
+# -*- coding: utf-8 -*-
+
+__author__ = 'Olav Espenes'
+__email__ = 'olaves@nmbu.no'
+
+
+class LCGRand:
+    def __init__(self, seed):
+        self.seed = seed
+        self.a = 7**5
+        self.m = 2 ** 31 - 1
```

Comment:
These should be class variable. Please check the skeleton from task description.